### PR TITLE
rust/doc : add rust building

### PR DESCRIPTION
**Pull request by caicancai**
## Related issue number

Closes #xxxx

## Check code requirements

- [ ] tests added / passed (if needed)
- [ ] Ensure all linting tests pass, see [here](https://github.com/alipay/fury/blob/main/CONTRIBUTING.rst) for how to run them

### CODE REVIEW

**Comment by caicancai** [comment]
The DEVELOPMENT.md layout seems a little ugly, I will re-optimize it later

**Comment by chaokunyang** [comment]
@wangweipeng2 Could you help review this?

File: docs/guide/DEVELOPMENT.md [hunk 1]
```diff
@@ -67,3 +67,19 @@ go test -v fury_xlang_test.go
 #### Environment Requirements
 
 - go1.3+
+
+### Building Fury Rust
+
+```bash
+cd rust/fury
+# build
+rust build
```
Comment (wangweipeng2) [inline docs/guide/DEVELOPMENT.md:76]: Rust usually uses Cargo to build and test, is this a typo?

**Review by wangweipeng2 — APPROVED** [review]
LGTM

**Review by wangweipeng2 — CHANGES_REQUESTED** [review]
@caicancai  "cd rust' is ok. The root directory is a cargo workspace, and cargo would automatically lookup the members."

**Comment by caicancai** [comment]
a15a906b6bb1e373bca3f5625e2292f

cd rust, and then execute cargo test will give an error

**Comment by wangweipeng2** [comment]
> a15a906b6bb1e373bca3f5625e2292f
> cd rust, and then execute cargo test will give an error. cd rust/fury will not report an error. cc @chaokunyang
@caicancai 
It should have been caused by my incorrect code the day before yesterday, so the error report is reasonable. I will fix it.

**Review by wangweipeng2 — APPROVED** [review]
LGTM

**Comment by chaokunyang** [comment]
> a15a906b6bb1e373bca3f5625e2292f
> cd rust, and then execute cargo test will give an error. cd rust/fury will not report an error. cc @chaokunyang

Good catch, thanks for this, @caicancai

**Pull request merged**